doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-business-rules-task-proc.adoc

```diff
@@ -18,7 +18,7 @@ Business rules tasks are used to make decisions through a Decision Model and Not
 |Description
 
 | *Name*
-| The name of the business rule task.
+| The name of the business rule task. You can also double-click the shape to edit the name of the business rule task. 
```

Comment:
Can you please elaborate which shape we have to double-click? Is this a text field?

Reply:
Have changed to 
You can also double-click the business rule task shape to edit the name.

for all the related occurrences.